mne_bids/tsv_handler.py

```diff
@@ -44,6 +44,51 @@ def _combine(data1, data2, drop_column=None):
     return data
 
 
+def _combine_cols(data1, data2, key_order, col_name):
+    """Add two OrderedDict's together and optionally drop repeated data.
+
+    Parameters
+    ----------
+    data1 : collections.OrderedDict
+        Original OrderedDict.
+    data2 : collections.OrderedDict
+        New OrderedDict to be added to the original.
+    key_order : list
```

Comment:
why do you need to pass this?

Reply:
Sorry, this was definitely named improperly. The correct name would be `skip_keys`, which allows one to skip a set of the keys in the OrderedDict (i.e. columns in the participants.tsv file)

This list is needed so that the columns in participants.tsv are not overwritten with the old data if they were updated in the current run.  For example:

If original participants.tsv OrderedDict (i.e. data1) read in from the file was:

```
{participant_id = ['01', '02'], age=['n/a', 'n/a'], ... testcol=['n/a', 43]}
```

and the new participants.tsv OrderedDict formed by mne-bids was:

```
{participant_id = ['01', '02'], age=['n/a', 20], ... testcol=['n/a', 'n/a']}
```

Then this parameter is needed so that way age isn't overwritten and just testcol is.